boards/arm/lpcxpresso55s69/lpcxpresso55s69_ns_defconfig

```diff
@@ -19,6 +19,9 @@ CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=96000000
 # TFM sets up MPU_NS, can't correctly change this configuration yet
 CONFIG_ARM_MPU=n
 
+# TF-M limits the IAP flash controller to secure-only
+CONFIG_SOC_FLASH_MCUX=n
+
```

Comment:
Instead of deselecting this in the defconfig should we instead have `CONFIG_SOC_FLASH_MCUX` default to `n` if `CONFIG_TRUSTED_EXECUTION_NONSECURE=y`?

Reply:
Fixed, thanks.